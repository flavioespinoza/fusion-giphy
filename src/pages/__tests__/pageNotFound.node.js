// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import PageNotFound from '../pageNotFound';

it('PageNotFound renders correctly', () => {
  const tree = renderer
    .create(
      <PageNotFound>
        <div>404</div>
      </PageNotFound>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
