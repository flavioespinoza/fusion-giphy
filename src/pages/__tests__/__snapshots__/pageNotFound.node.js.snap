// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound renders correctly 1`] = `
<div>
  404
</div>
`;
